linkedin_llm_agent/prompts.py

```diff
@@ -180,4 +180,35 @@
     The post usually can't have 10/10 points because something is not perfect.
 
 Post content to evaluate:
+"""
+
+
+extracting_key_features_from_link_info = """
```

Comment:
naming suggestion: `get_key_features_from_user_info_prompt` or something like that. Idea is to use a shorter word (get) and the extraction is not specific to LinkedIn (any user info will work)